processor/src/main/resources/org/mapstruct/ap/internal/model/assignment/ArrayCopyWrapper.ftl

```diff
@@ -20,11 +20,11 @@
 -->
 <#if (thrownTypes?size == 0) >
     <@includeModel object=ext.targetType/> ${localVarName} = <@_assignment/>;
-    ${ext.targetBeanName}.${ext.targetWriteAccessorName}( Arrays.copyOf( ${localVarName}, ${localVarName}.length ) );
+    ${ext.targetBeanName}.${ext.targetWriteAccessorName}${afterWriteVariable}Arrays.copyOf( ${localVarName}, ${localVarName}.length )${finishWriteStatement};
```

Comment:
@sjaakd These are the points which I don't really like in my PR. The getters are similar.